.github/workflows/release-benchmarks.yml

```
@@ -0,0 +1,93 @@
+          fetch-depth: 0
+          persist-credentials: false
+          ref: ${{ github.event.release.tag_name }}
+
+      - name: Install Rust toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@166cdcfd11aee3cb47222f9ddb555ce30ddb9659 # v1.17.0
+        with:
+          cache: false
+          cache-bin: false
+
+      - name: Set up just
+        uses: $/.github/actions/setup-just
```

actionlint (1.7.12)

[error] specifying action "$/.github/actions/setup-just" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)